widgetssdk/src/main/java/com/glia/widgets/view/Dialogs.java

```diff
@@ -33,9 +33,17 @@ public static AlertDialog showOptionsDialog(Context context,
                                                 View.OnClickListener positiveButtonClickListener,
                                                 View.OnClickListener negativeButtonClickListener,
                                                 DialogInterface.OnCancelListener cancelListener) {
+
+        boolean isUseAlertDialogButtonVerticalAlignment = Utils.getGliaAlertDialogButtonUseVerticalAlignment(theme);
```

Comment:
this layout code is duplicated 3 times in the class. can you separate it into a method like `getLayout()`?

Reply:
different layouts

it will be getLayout(boolean, resid, resid) not sure if worth it